README.md

```diff
@@ -55,7 +55,9 @@ These can be installed with `dpkg -i summon_v*.deb` and
 
 ### Auto Install
 
-**Note** Check the release notes and select an appropriate release to ensure support for your version of Conjur.
+**Note** Check the 
```

Comment:
There's a trailing space just after "Check the".
Same on next line, there's a trailing space just after "version of Conjur."

Reply:
OK, I thought they were needed. I have removed them.